client/src/layout/SortToggle.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+
+const group = {
+  backgroundColor: "#f0f1fa",
+  borderRadius: "50px",
+  padding: "5px",
+};
+
+const selected = {
+  backgroundColor: "#657DF2",
```

Comment:
a kind reminder that we have some most used colour set in theme, you could just use them. There's also spacing defined in Material-UI

Reply:
I ran into lots of trouble with conditional styling when using makeStyles. That's why I ended up not using it and took this approach instead. I used the primary hex code for the styling.